src/charm.py

```diff
@@ -57,6 +57,9 @@ def __init__(self, *args):
         self.framework.observe(self.on.nms_pebble_ready, self._configure_sdcore_nms)
         self.framework.observe(self.on.update_status, self._configure_sdcore_nms)
         self.framework.observe(self.fiveg_n4.on.fiveg_n4_available, self._configure_sdcore_nms)
+        self.framework.observe(
+            self.on.sdcore_management_relation_joined, self._configure_sdcore_nms
```

Comment:
How could this have caused the problem? We already observe `secore_management_url_available` which would be triggered after the relation joined.

Reply:
Right, I also found this strange.  We got following error in the logs:
```
unit-nms-0: 2024-09-23 08:59:15 ERROR juju.worker.uniter.operation hook "sdcore-management-relation-changed" (via hook dispatching script: dispatch) failed: exit status 1
unit-nms-0: 2024-09-23 08:59:15 DEBUG juju.machinelock created rotating log file "/var/log/juju/machine-lock.log" with max size 10 MB and max backups 5
unit-nms-0: 2024-09-23 08:59:15 DEBUG juju.machinelock machine lock "machine-lock" released for nms/0 uniter (run relation-changed (20; app: webui) hook)
unit-nms-0: 2024-09-23 08:59:15 DEBUG juju.worker.uniter.operation lock released for nms/0
unit-nms-0: 2024-09-23 08:59:15 DEBUG juju.worker.uniter.actions no next action from pending=[]; completed=map[]
unit-nms-0: 2024-09-23 08:59:15 INFO juju.worker.uniter awaiting error resolution for "relation-changed" hook
unit-nms-0: 2024-09-23 08:59:15 DEBUG juju.worker.uniter [AGENT-STATUS] error: hook failed: "sdcore-management-relation-changed"
```

Reply:
The real reason is found which causes failure in sdcore-management-relation changes. 
NMS image does not include GNB and UPF config paths any more as described in the PR definition.